frontend/src/metabase/ui/docs/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "docs",
```

Comment:
nit: Maybe "Metabase UI documentation" to be more specific?

Reply:
Oh sure, I think it just grabbed the the directory name when I initialized the project.